engine/apps/alerts/tests/integration_test_alert_flow.py

```diff
@@ -0,0 +1,69 @@
+import time
+from uuid import uuid4
+
+import pytest
+
+from apps.alerts.models import AlertGroup, AlertReceiveChannel
+from apps.integrations.tasks import create_alert
+
+pytest_plugins = ["celery.contrib.pytest"]
+
+# Uncomment this block if you want to test specific queues
+# @pytest.fixture(scope='session')
+# def celery_worker_parameters():
+#     return {
+#         'queues':  ('celery', 'default', 'retry', 'critical'),
+#     }
+
+
+@pytest.fixture(scope="session")
```

Comment:
`scope="session"` means that this fixture is auto-used once for **all** tests, correct? could this maybe be scoped to this file (`scope="module"`)? or if it makes sense to keep it to `scope="session"`, maybe move it to `engine/conftest.py`?